### PR TITLE
changes to support Jakartaee10 jars and Glassfish 7

### PR DESCRIPTION
**Pull request by gurunrao**
changes to support Jakartaee10 jars and Glassfish 7.

### CODE REVIEW

**Comment by starksm64** [comment]
In the #12 PR is have reorganized the artifacts and modules to be generic, and the version reflects the GlassFish version. Once this PR is merged, I can update the master branch to use the same organization.

**Review by starksm64 — CHANGES_REQUESTED** [review]

File: glassfish-common/pom.xml
Comment (gurunrao) [reply glassfish-common/pom.xml:4]: There is an overlap in changes made by current PR and PR #12 (which was open post PR #11), example similar changes include making package generic from glassfish_6 to glassfish,  I will wait for PR#12 to be merged and resolve conflicts post that.

File: pom.xml
Comment (gurunrao) [reply pom.xml:32]: Based on https://repo1.maven.org/maven2/org/jboss/arquillian/container/arquillian-container-spi/ shall we use `1.7.0.Alpha9`

**Comment by gurunrao** [comment]
PR #12  needs to be merged and conflicts needs to be resolved before PR can be merged.

**Comment by starksm64** [comment]
If I try to build your fork with SE 11, I'm seeing that the org.glassfish.hk2:hk2-locator:jar:3.0.0-M1 artifact cannot be found:

```
[ERROR] Failed to execute goal on project arquillian-glassfish6-common: Could not resolve dependencies for project org.jboss.arquillian.container:arquillian-glassfish6-common:jar:1.0.0.Final-SNAPSHOT: org.glassfish.hk2:hk2-locator:jar:3.0.0-M1 was not found in https://jakarta.oss.sonatype.org/content/repositories/staging/ during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of jakarta-nexus-staging has elapsed or updates are forced -> [Help 1]
```

**Comment by gurunrao** [comment]
> If I try to build your fork with SE 11, I'm seeing that the org.glassfish.hk2:hk2-locator:jar:3.0.0-M1 artifact cannot be found:
> 
> ```
> [ERROR] Failed to execute goal on project arquillian-glassfish6-common: Could not resolve dependencies for project org.jboss.arquillian.container:arquillian-glassfish6-common:jar:1.0.0.Final-SNAPSHOT: org.glassfish.hk2:hk2-locator:jar:3.0.0-M1
> was not found in https://jakarta.oss.sonatype.org/content/repositories/staging/ during a previous attempt. This failure was
> cached in the local repository and resolution is not reattempted until the update interval of jakarta-nexus-staging has
> elapsed or updates are forced -> [Help 1]
> ```
currently not able to reproduce the issue, attaching [logs](https://github.com/arquillian/arquillian-container-glassfish6/files/8477904/acg_build.log)

File: glassfish-common/pom.xml
Comment (gurunrao) [reply glassfish-common/pom.xml]: done.

**Comment by gurunrao** [comment]
> failure

^ failure is not reproducible at CI run https://ci.eclipse.org/jakartaee-tck/job/guru/job/cditck-porting-guru/job/jakartaee-10/81/consoleText

**Comment by gurunrao** [comment]
@starksm64  - `org.glassfish.hk2:hk2-locator:jar:3.0.0-M1 artifact cannot be found`  error is not seen in local setup or CI.

**Comment by gurunrao** [comment]
@starksm64  - Let me know if the PR can be merged, as review comments are not open?

**Review by starksm64 — APPROVED** [review]

**Comment by starksm64** [comment]
I merged it. We still need to get the main branch CI config updated and working.

**Pull request merged**
